setup.py

```diff
@@ -4,18 +4,18 @@
 import subprocess
 
 setup(name="singer-python",
-      version='6.0.0',
+      version='6.0.1',
       description="Singer.io utility library",
       author="Stitch",
       classifiers=['Programming Language :: Python :: 3 :: Only'],
       url="http://singer.io",
       install_requires=[
           'pytz>=2018.4',
           'jsonschema==2.6.0',
-          'simplejson==3.11.1',
+          'simplejson>=3.11.1,==3.*',
           'python-dateutil>=2.6.0',
-          'backoff==2.2.1',
-	  'ciso8601',
+          'backoff>=2.2.1,==2.*',
+          'ciso8601',
```

Comment:
is this whitespace, and if so, can we remove it?

Reply:
It was a using a tab. I changed it to spaces.